jumpssh/session.py

```diff
@@ -222,7 +223,8 @@ def run_cmd(
             success_exit_code=0,
             retry=0,
             retry_interval=5,
-            keep_retry_history=False
+            keep_retry_history=False,
+            get_pty=False
```

Comment:
For backward compatibility purpose, the default value for get_pty should be True